### PR TITLE
fix(crux-ui): version hint

### PR DESCRIPTION
**Pull request by peterambrus3**
fix(crux-ui): Version semantic hint
fix(crux-ui): Large gap on the right side of the main layout
fix(crux-ui): Not consistent breadcrumb header positions

### CODE REVIEW

File: web/crux-ui/src/components/products/versions/edit-version-card.tsx [hunk 1]
```diff
@@ -81,6 +81,21 @@ const EditVersionCard = (props: EditVersionCardProps) => {
     submitRef.current = formik.submitForm
   }
 
+  const checkVersionHint = (value: string) => {
+    const err = getValidationError(nameRule, value)
+    if (!versionRegex.test(value) && !err) {
```
Comment (robot9706) [inline web/crux-ui/src/components/products/versions/edit-version-card.tsx:86]: Wouldn't be easier to use yup to validate the regex?

**Review by robot9706 — CHANGES_REQUESTED** [review]

File: web/crux-ui/locales/en/versions.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "versionsName": "Products - {{product}} - Versions - {{name}}",
 
-  "tips": "Create a new version for your product that you can use for different purposes. Rolling versions are ideal for continuous delivery, while Incremental ones are useful for production.",
+  "tips": "Create a new version for your product. You can select to default a version other versions will inherit images and configurations from. dyrector.io is the most suitable for semantic versioning, since it's the de facto way of version control. We recommend using that.",
```
Comment (robot9706) [inline web/crux-ui/locales/en/versions.json:4]: `You can select a default version, from which other versions will inherit images and configuration` ?

**Review by robot9706 — CHANGES_REQUESTED** [review]

File: web/crux-ui/locales/en/versions.json (hunk 2 above)
Comment (peterambrus3) [reply web/crux-ui/locales/en/versions.json:4]: i just followed the instructions, but i like your version

Comment (polaroi8d) [reply web/crux-ui/locales/en/versions.json:4]: @robot9706 suggestion looks better for me.

**Review by robot9706 — APPROVED** [review]
LGTM 😎

**Review by nandor-magyar — APPROVED** [review]

**Pull request merged**